pkg/services/validator/maas.go

```diff
@@ -220,14 +208,6 @@ func readMaasResourceRule(c *components.MaasPluginConfig, r *vpmaasapi.ResourceA
 		return err
 	}
 
-	if r.Name == "" {
-		name, err := prompts.ReadText("Rule Name", "", false, -1)
-		if err != nil {
-			return err
-		}
-		r.Name = name
-	}
-
 	if r.AZ == "" {
 		az, err := prompts.ReadText("Availability Zone", "az1", false, -1)
```

Comment:
Sorry Artur - I missed this one yesterday. Can you please provide a selection for AZs as well?